e2e/tsconfig-paths/tsconfig-paths.e2e.ts

```diff
@@ -2,9 +2,13 @@ import { createTenv, getAvailablePort } from '@internal/e2e';
 import { fetch } from '@whatwg-node/fetch';
 import { expect, it } from 'vitest';
 
-const { gateway, fs } = createTenv(__dirname);
+const { gateway, fs, gatewayRunner } = createTenv(__dirname);
 
-it('should start gateway', async () => {
+const skipIfIt = function (condition: boolean) {
+  return condition ? it.skip : it;
+};
+
+skipIfIt(gatewayRunner.includes('bun'))('should start gateway', async () => {
```

Comment:
This is not necessary, [vitest supports `it.skipIf`](https://vitest.dev/api/#test-skipif).

Reply:
Done!